libcontainer/configs/cgroup_linux.go

```diff
@@ -32,7 +32,7 @@ type Cgroup struct {
 	*Resources
 
 	// Systemd tells if systemd should be used to manage cgroups.
-	Systemd bool
+	Systemd bool `json:"systemd"`
```

Comment:
Wouldn't this already work though? (only using `Systemd` (capital S) when marshalling? ) https://go.dev/play/p/uVMT3i5XvDw

Reply:
Sorry for delay reply. It is my mistake to mess up the version。